customers/views.py

```diff
@@ -123,9 +119,9 @@ def update(customer_id):
                 "email": f"{updated_customer.email}",
                 "dob": f"{updated_customer.dob}",
                 "date_created": f"{updated_customer.date_created}",
-            }
+            }, 200
     except Exception:
-        return "ERROR, Code: 404, invalid id number entered"
+        return {"Message": "ERROR, invalid id number entered"}, 404
 
 
 @views.route("/delete/<int:customer_id>", methods=["DELETE"])
```

Comment:
same here, I should be able to call DELETE at the path `/customers/1` (if the ID is 1)